src/utils/i18n-es-en.ts

```diff
@@ -0,0 +1,68 @@
+const translations: { [key: string]: string } = {
+  "Visual Breathing Exercise For Stress & Anxiety":
+    "Ejercicio de respiración visual para el estrés y la ansiedad",
+  "Set your custom duration": "Establece tu duración personalizada",
+  "Enter number of cycles": "Introduce el número de ciclos",
+  Start: "Iniciar",
+  Custom: "Personalizado",
+  Donate: "Donar",
+  "Quick Exit": "Salida rápida",
+  "Press the ESC button three times to quickly leave our site":
```

Comment:
I would like to recommendo adding keywords inteads of the full text, the creating the the translation for both languages such as:

es.ts
```
{
"press-esc": "Presiona el botón ESC tres veces para salir rápidamente de nuestro sitio"
}
```

en.ts
```
{
"press-esc": ""Press the ESC button three times to quickly leave our site"
}
```

it will help us to expand the app in the future and support more languages.